app/vmselect/promql/alias.go

```diff
@@ -0,0 +1,9 @@
+package promql
+
+import (
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/promql"
+)
+
+// DurationValue returns the duration in milliseconds for the given s
+// and the given step.
+var DurationValue = promql.DurationValue
```

Comment:
Just reexporting this from the `lib/promql` package since it's used elsewhere in vmselect